synapse/lib/aha.py

```diff
@@ -682,8 +693,126 @@ def _initCellHttpApis(self):
         self.addHttpApi('/api/v1/aha/services', AhaServicesV1, {'cell': self})
         self.addHttpApi('/api/v1/aha/provision/service', AhaProvisionServiceV1, {'cell': self})
 
+    async def callAhaSvcApi(self, name, todo, timeout=None):
```

Comment:
Looks like this was replaced with `callAhaPeerApi`?

Reply:
The use case is a little different. The peer one automatically looks up peers for an iden. This API is designed to target a specific service via AHA name :)